### DISCUSSION TITLE
Problem with exporting formula to non-English Excel

### DISCUSSION THREAD

**Discussion by Buggy001** [dfinke/ImportExcel#1584] [Q&A]
Hello,

I use a non-English version of Excel.
This seems to give problems when exporting formulas. I get the 'We found a problem with some content in' error message when I open the sheet.

I do this:
$FilePath = "D:\support\test\test001.xlsx"
$sh = 0..5 | Export-Excel $FilePath -WorksheetName 'Lijst' -AutoSize -Passthru
This works, and there is 0 to 5 in the first column.

When I change the cells to a formula, a simple sum as an example, I get the error message above:
$AFormula = '==SOM(1;2)'
Set-ExcelRange -Range $sh.Workbook.Worksheets["Lijst"].Cells["A2:A6"]  -Formula $AFormula
$sh.save()
And the cells are empty.

When I change the formula to:
$AFormula = '==SOM(1,2)'
And I do the same again. This time, there is no error message and the formula has been inserted. The comma is automatically converted to a semicolon. And the formula works.

Why does this go wrong? And how can I solve it? So that I can use a semicolon, as it should be in my Excel version.
Thanks!

With kind regards,

Jac

**Comment by scriptingstudio** [comment]
A localized Excel/Word has its own function names. You have to have a dictionary to translate eng to non-eng names or to have them hardcoded.
